src/js/mozilla.cookies.js

```diff
@@ -34,9 +34,11 @@ if (typeof Mozilla === 'undefined') {
 Mozilla.Cookies = {
     getItem: function (sKey) {
         if (!sKey) { return null; }
+        /* eslint-disable-next-line no-useless-escape */
```

Comment:
Note: I didn't want to go start making linting changes to the code, so I just ignored these errors for now. It may also be worth splitting libs like this into their own sub-folder perhaps, so ESLint can more easily ignore externally maintained scripts?

Reply:
I think this is the right amount of change for this PR 👍